matlab/src/feather_writer.h

```diff
@@ -0,0 +1,76 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#ifndef MLARROW_FEATHER_WRITER_H
+#define MLARROW_FEATHER_WRITER_H
+
+#include <memory>
+#include <string>
+
+#include <arrow/ipc/feather.h>
+#include <arrow/status.h>
+#include <arrow/type.h>
+
+#include <matrix.h>
+
+namespace mlarrow {
```

Comment:
You could spare yourself a lot of typing in this codebase by nesting the matlab symbols in a namespace inside `arrow`, e.g. `namespace arrow { namespace matlab {` (or whatever NS name you need to use to avoid any symbol conflicts)

Reply:
Fixed.